packages/shared/src/modules/users/libs/validation-schemas/user-sign-up.validation-schema.ts

```diff
@@ -13,7 +13,7 @@ const userSignUp = z
 		email: z
 			.string()
 			.trim()
-			.min(UserValidationRule.EMAIL_MINIMUM_LENGTH, {
+			.min(UserValidationRule.EMAIL_REQUIRE, {
```

Comment:
This shouldn't have changed

Reply:
UserValidationRule.EMAIL_REQUIRE does not exist on the main branch